File: Java/catalogovideos/src/main/java/br/com/iteris/universidade/catalogovideos/domain/entity/Video.java
package br.com.iteris.universidade.catalogovideos.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.util.List;

@Data
@Entity
@NoArgsConstructor
@AllArgsConstructor
public class Video {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int idVideo;
    public String titulo;
    public String url;

    @OneToMany(mappedBy = "video", fetch=FetchType.LAZY)
    private List<VideoCategoria> videoCategorias;

    @OneToOne(fetch=FetchType.LAZY)
    @JoinColumn(name="idResponsavel")
    private Responsavel responsavel;

}

File: C#/CatalogoVideos/CatalogoVideos/Domain/Entity/Video.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CatalogoVideos.Domain.Entity
{
    public class Video
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public int ResponsavelId { get; set; }
        [Required]
        [StringLength(255)]
        public string Titulo { get; set; }
        [Required]
        [StringLength(255)]
        public string URL { get; set; }
        public int? Idade { get; set; }


        public virtual Responsavel Responsavel { get;set;}
        public virtual ICollection<VideoCategoria> VideoCategorias { get; set; }
    }
}
